recipes/asciitree/meta.yaml

```diff
@@ -0,0 +1,39 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "0.3.2" %}
+{% set sha256 = "eb10abcd9b079b905bb7b4b23bb030331e8aa1ab76a9a1323bd43a9d3138f0e7" %}
+
+package:
+  name: asciitree
+  version: {{ version }}
+
+source:
+  fn: asciitree-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/a/asciitree/asciitree-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  number: 0
```

Comment:
This change caused appveyor to break. Will this be a problem for the inclusion of the package? 
I haven't tested it myself on windows but I don't see anything in it's code suggesting why it shouldn't work just fine there as well.